### PR TITLE
Add syncStatus and lastSyncError to SrivoNetworkNodeState CR

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

File: manifests/4.2/sriov-network-operator-sriovnetwork.crd.yaml [hunk 1]
```diff
@@ -44,6 +40,8 @@ spec:
               type: boolean
             trust:
               type: boolean
+              minimum: 0
```
Comment (zshi-redhat) [inline manifests/4.2/sriov-network-operator-sriovnetwork.crd.yaml]: why adding min/max for boolean type value?

File: manifests/4.2/sriov-network-operator-sriovnetworknodestate.crd.yaml [hunk 2]
```diff
@@ -29,6 +29,8 @@ spec:
           type: object
         spec:
           properties:
+            dpConfigVersion:
```
Comment (zshi-redhat) [inline manifests/4.2/sriov-network-operator-sriovnetworknodestate.crd.yaml]: I didn't see this is used anywhere `dpConfigVersion`, what should be the value of it?

Comment (pliurh) [reply manifests/4.2/sriov-network-operator-sriovnetworknodestate.crd.yaml]: It is not related to this patch, but a update to make sure the crds in `manifests` directory is the same as the ones in `deploy`

File: manifests/4.2/sriov-network-operator-sriovnetwork.crd.yaml (hunk 1 above)
Comment (pliurh) [reply manifests/4.2/sriov-network-operator-sriovnetwork.crd.yaml]: It is a error caused by rebase, I'll fix it

**Pull request merged**